### DISCUSSION TITLE
Remove protected setter for InputProperties in BusinessRuleBase

### DISCUSSION THREAD

**Discussion by StefanOssendorf** [MarimerLLC/csla#3855] [Ideas]
As the title says:
Remove the protected setter for InputProperties. It should be initialized by the base class and users should only add to it not replace it.
If they want to replace the content they can clear and add to do that. This would make the future "null handling" also easier.

**Comment by rockfordlhotka** [comment]
Shouldn't this be an issue on the backlog?

**Reply by StefanOssendorf** [reply]
I wanted to propose it here first and now create the issue :D
